src/overview.md

```diff
@@ -240,6 +240,22 @@ More information:
 * [`webdev` tool](/tools/webdev)
 * [Web deployment tips](/web/deployment)
 
+#### The Dart Runtime {#runtime}
+
+Regardless of how you compile your Dart code, a Dart runtime is needed to
+execute the code. This runtime is responsible for a number of critical tasks:
+
+* Managing memory: Dart uses a managed memory model, where unused memory is
+  reclaimed by a garbage collector (GC). 
+
+* Enforcing the Dart type system: While most type checks are static (compile
+  time), some type checks are dynamic (run time), e.g. the Dart
+  [type check and cast operators](/guides/language/language-tour#type-test-operators).
+
+On the native platforms, the Dart runtime is automatically included inside
+self-contained executables, and is part of the Dart VM invoked with [`dart
+run`](https://dart.dev/tools/dart-run). On the web platforms, the runtime is
```

Comment:
The part about runtime on a Web is a bit confusing, e.g. GC is provided by the JavaScript runtime. I'd remove the last sentence to avoid confusing people too much.

Reply:
But isn't there still a Dart runtime when running on the web?

There was a confusion recently where someone asked why they could not run the output of `dart compile js` in node, which I _think_ boiled down to having to include some JS code that essentially is the runtime?

Reply:
Well, there are some parallel, but calling it runtime carries some connotations that I think we would like to avoid (slow, heavy, foreign). Yes, there is some code that needs to run to bootstrap things, some code that implements type checks and some stuff for deferred loading. That's about it.

There are differences between how code runs in node vs browser (differences in global object, module system, etc) hence some challenges in running `dart2js` code in node... But you would have similar issues with any other JavaScript which is not entirely platform agnostic.

Reply:
OK, I'll remove the runtime part from web than.